pkg/noun/allocate.c

```diff
@@ -2094,45 +2104,131 @@ u3a_prof(FILE* fil_u, c3_w den_w, u3_noun mas)
 #endif
       }
 #endif
+      pro_u->nam_c = u3r_string(h_mas);
+      pro_u->siz_w = siz_w*4;
+      pro_u->len_w = 0;
+      pro_u->qua_u = 0;
+      return pro_u;
 
-      return tot_w;
     }
     else if ( c3n == it_mas ) {
       fprintf(fil_u, "\r\n");
```

Comment:
This can be removed now, otherwise we get tons of newlines printed before the memory measurements.